bin/node/cli/tests/temp_base_path_works.rs

```diff
@@ -29,37 +29,36 @@ use std::{
 	io::Read,
 	path::PathBuf,
 	process::{Command, Stdio},
-	thread,
-	time::Duration,
 };
 
 pub mod common;
 
-#[test]
-fn temp_base_path_works() {
+#[tokio::test]
+async fn temp_base_path_works() {
+	// Test depends on log output so set RUST_LOG:
 	let mut cmd = Command::new(cargo_bin("substrate"));
-
-	let mut cmd = cmd
-		.args(&["--dev", "--tmp"])
-		.stdout(Stdio::piped())
-		.stderr(Stdio::piped())
-		.spawn()
-		.unwrap();
+	let mut child = common::KillChildOnDrop(
+		cmd.args(&["--dev", "--tmp"])
+			.env("RUST_LOG", "info")
```

Comment:
```suggestion
```

You don't need this. I tried your local branch without this and it worked.

Reply:
Agreed. I see the same now.